torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -1,37 +1,42 @@
 from __future__ import annotations
 
 import inspect
-
 import operator
-
 import re
 import types
 import warnings
-
 from typing import Callable, Dict, List, Optional, Sequence, Tuple, Union
 
 import onnxscript  # type: ignore[import]
-from onnxscript import evaluator, opset18  # type: ignore[import]
-from onnxscript.function_libs.torch_lib import graph_building  # type: ignore[import]
+from onnxscript.function_libs.torch_lib import (  # type: ignore[import]
+    graph_building as onnxscript_graph_building,
+)
 
 import torch
-import torch.fx
 from torch._subclasses import fake_tensor
-
 from torch.onnx import _type_utils
 from torch.onnx._internal import _beartype
-from torch.onnx._internal.exporter import ResolvedExportOptions
-from torch.onnx._internal.fx import diagnostics, op_validation
+from torch.onnx._internal.fx import diagnostics, onnxfunction_dispatcher, op_validation
 from torch.utils import _pytree
 
 
 @_beartype.beartype
 def _fx_node_to_onnx_message_formatter(
     fn: Callable,
-    diagnostic_context: diagnostics.DiagnosticContext,
+    self,
     node: torch.fx.Node,
-    *args,
-    **kwargs,
+    fx_graph_module: torch.fx.GraphModule,
```

Comment:
why expand on the varargs?

Reply:
I though a perfect match was required. if it is not, I can revert this and make things shorter

Reply:
no it's not, only the ones used are recommended to be expanded.

Reply:
pushing new revision now